### ISSUE TITLE
Soft Lockup - CPU#0 stuck for x

### ISSUE THREAD

**Issue by CacheZero1** [utmapp/UTM#5360]
Hi, I have been trying to get Metasploitable to work on my M2 Macbook air. Everytime I try to run the vm tho, it tells me `BUG: Soft Lockup - CPU#0 stuck for x s` after the Getting ready prompt.

**Configuration**  
* UTM Version: 4.2.5 (81)
* macOS Version: Ventura 13.3.1
* Mac Chip: M2 

**Debug log**  
`Running:  -L /Applications/UTM.app/Contents/Resources/qemu -S -spice "unix=on,addr=/Users/jones/Library/Group Containers/WDNLXAD4W8.com.utmapp.UTM/859F44D9-56DD-4648-A76A-C204BABC063E.spice,disable-ticketing=on,image-compression=off,playback-compression=off,streaming-video=off,gl=off" -chardev spiceport,id=org.qemu.monitor.qmp,name=org.qemu.monitor.qmp.0 -mon chardev=org.qemu.monitor.qmp,mode=control -nodefaults -vga none -device e1000,mac=B6:A7:25:E4:00:34,netdev=net0 -netdev vmnet-host,id=net0 -device virtio-vga -smp cpus=1,sockets=1,cores=1,threads=1 -machine q35,vmport=off -accel tcg,tb-size=1024 -global PIIX4_PM.disable_s3=1 -global ICH9-LPC.disable_s3=1 -m 4096 -audiodev coreaudio,id=audio0 -device intel-hda -device hda-duplex,audiodev=audio0 -usb -device usb-tablet,bus=usb-bus.0 -device usb-mouse,bus=usb-bus.0 -device usb-kbd,bus=usb-bus.0 -device nec-usb-xhci,id=usb-controller-0 -chardev spicevmc,name=usbredir,id=usbredirchardev0 -device usb-redir,chardev=usbredirchardev0,id=usbredirdev0,bus=usb-controller-0.0 -chardev spicevmc,name=usbredir,id=usbredirchardev1 -device usb-redir,chardev=usbredirchardev1,id=usbredirdev1,bus=usb-controller-0.0 -chardev spicevmc,name=usbredir,id=usbredirchardev2 -device usb-redir,chardev=usbredirchardev2,id=usbredirdev2,bus=usb-controller-0.0 -device ide-hd,bus=ide.0,drive=driveD3E2A72C-2E72-455A-9521-EEB537AC9215,bootindex=0 -drive "if=none,media=disk,id=driveD3E2A72C-2E72-455A-9521-EEB537AC9215,file=/Users/jones/Library/Containers/com.utmapp.UTM/Data/Documents/Metasploitable 2.utm/Data/MetasploitableM1.qcow2,discard=unmap,detect-zeroes=unmap" -device virtio-serial -device virtserialport,chardev=org.qemu.guest_agent,name=org.qemu.guest_agent.0 -chardev spiceport,id=org.qemu.guest_agent,name=org.qemu.guest_agent.0 -device virtserialport,chardev=vdagent,name=com.redhat.spice.0 -chardev spicevmc,id=vdagent,debug=0,name=vdagent -device virtserialport,chardev=charchannel1,id=channel1,name=org.spice-space.webdav.0 -chardev spiceport,name=org.spice-space.webdav.0,id=charchannel1 -name "Metasploitable 2" -uuid 859F44D9-56DD-4648-A76A-C204BABC063E -rtc base=localtime -device virtio-rng-pci
audio: Failed to create voice 'adc'
qemu-x86_64-softmmu: warning: global PIIX4_PM.disable_s3=1 not used`

**Comment by Teloshav** [comment]
Also have been having this exact issue.

Followed the steps from a comment on a Reddit post linked [here](https://reddit.com/r/metasploit/comments/pqy7dv/metasploitable_for_m1_mac/):

> 1. Set up homebrew if you haven't already.
> 2. Install qemu by running this in a terminal:
> 
> ```    
> brew install qemu
> ```
> 
> 3. Download UTM and install it by putting it in your applications.
> 4. Download Metasploitable from rapid7 and extract the zip somewhere.
> 5. Run this command in a terminal from the directory you extracted MS to. It will convert the vmdk disk image to qcow2.
> ```
> qemu-img convert -O qcow2 -c Metasploitable.vmdk Metasploitable.qcow2
> ```
> 6. In UTM, create a new VM. Chose "Emulate," "Other," "Skip ISO boot."
> 7. For hardware, leave it as is. Lower the RAM if you want, it only needs like 512M (maybe less).
> 8. For the rest of the options just next, next, next... until you get to the "summary" page.
> 9. On the summary page, select "open vm settings" then "save."
> 10. The VM settings will open. You can change the name at the top if you want.
> 11. Go to "qemu" in the settings and uncheck "UEFI Boot."
> 12. If you want the VM "on the network" with its own (probably) 192.168... address, go to "network" in the settings and select "bridged."
> 13. Under "drives" select "IDE Drive" and delete it.
> 14. Click new drive -> import, and select the qcow2 file you created in step 5.
> 15. Click save and start the VM.

Did a rough job here copying and pasting the comment contents.

This is what I followed along to and got this result:

![](https://github.com/utmapp/UTM/assets/131397191/764f8dc0-7335-4335-9e33-edc1135c583d)

**Comment by jckxr** [comment]
I have exact issue

**Comment by jckxr** [comment]
I was able to resolve my issue. the issue is coming from ram allocation.. ensure the total ram allocated to vms on your utm are not totaling to the exact ram on your host machine, that should solve it.

**Comment by Teloshav** [comment]
Yes I can confirm that the same solution worked for me, I figured this out soon after posting this from a colleague.

RAM: 1024 MB or 1GB
CPU Cores: 1
Architecture: x86_64

- Teloshav

**Comment by marvpaul** [comment]
Reduced RAM from 4096 to 1024 solved the issue for me as well (even though I have 16 GB of ram and no other machine was running). Thanks!

**Comment by NihaoKangkang** [comment]
> Reduced RAM from 4096 to 1024 solved the issue for me as well (even though I have 16 GB of ram and no other machine was running). Thanks!

Worked. Thanks a lot.
